packages/website-router/titles-tags/config.ts

```diff
@@ -0,0 +1,8 @@
+import { TitleConfig } from './title-seo';
+
+export const MANAGE_TITLE: TitleConfig[] = [
+  {
+    titleSEO: 'GraphQL TypeScript',
```

Comment:
If that's the change we need to do, and it's very specific, why not do that in GraphQL Codegen repo?

Reply:
@dotansimha  
Oh, I added only one example. we have something like ~20 titles from different websites